cypress/support/commands.js

```diff
@@ -23,3 +23,23 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder = "${placeholder}"]`);
+  })
+
+Cypress.Commands.add('login', (username = 'B@ry!', password = 'B@ry!123') => {
```

Comment:
Consider using more descriptive default values for the 'username' and 'password' parameters, e.g. 'testUser' and 'testPassword123' 📝